### DISCUSSION TITLE
よくある質問ページを作る

### DISCUSSION THREAD

**Discussion by waxsd100** [TUSB/TheUnusualSkyBlock#306] [Idea]
死の宣告とはなんですか 、○○すると奈落に行きます）とかそういうページ作りたい

Q&Aを考えて

**Comment by Suyarumi** [comment]
Q.最初のゲートに入っても奈落に落とされてしまいます
A.主にコマンドブロックのクロックが動いていないと起こる仕様です。

お使いの環境及びマイクラのversionのご確認の上、
コマンドブロックが動く環境でプレイをして下さい
(MODが入っている場合サポート対象外です)

尚、マルチプレイの際はReadmeをお読み下さい。
また一度でも違うMinecraftバージョンでWorldにログインしますとWorldが破損します。
こんな感じで書けばいいのかな

**Comment by Orange2525** [comment]
Q.公式サイトのダウンロードボタンを押したのになぜかダウンロードされなかった。

A.ブラウザは最新バージョンでしょうか？最新バージョンでもダウンロードできない場合はお手数ですが、Google Chrome等の他のブラウザに変更してダウンロードしてみてください。

**Comment by Orange2525** [comment]
Q.「Forge」が入ってる状態でTUSBのマップに間違えて入ってしまったので、「Forge」を抜いてさっきのマップ入ったら「Forge」抜いたのにもかかわらず、マップがバグったままだった。

A.これはForge環境等で発生し、またForgeを抜いた後もこの効果は継続します。直す方法は現状なく、こうなってしまうともう一度、最初から導入し直し、バニラの状態でプレイしない限り直す事はできません。

**Comment by Orange2525** [comment]
Q.スポナーの位置に本来表示されるエフェクトが表示されない。またスキル発動時のエフェクトが表示されない。

A.「設定」→「ビデオ設定」→「パーティクルの表示設定」が最小だと表示されません、最小以上にしてみてください。

**Comment by Orange2525** [comment]
Q.実況動画で見た嫌な予感が発生したとき、画面が赤くならないです改善方法を教えてほしいです。

A.「設定」→「ビデオ設定」→「グラフィックス」を描画優先にしてください。

**Comment by Orange2525** [comment]
Q.エリトラや、速度上昇アイテムが正常に作動しません。

A.ワールドに入り直してみてください。

**Comment by Orange2525** [comment]
Q.質問ページに自分と同じような症状のQ＆Aがない。

A.The Unusual SkyBlock 報告と提案ページがございます、お手数ですがそこで質問してくと直接回答いたします、是非ご利用ください。（Discordの方にもございます、参加は手間ですが回答はこちらのほうが早いです。こちらも是非ご利用ください。。）

**Comment by waxsd100** [comment]
**Wikiに書いてある内容や既にあげられている内容のQ&Aも追加お願いします。**

**Comment by Orange2525** [comment]
Q.導入方法を教えてください。

A.こちら(Wikiのリンク)に載っています、御覧ください。

**Comment by Orange2525** [comment]
Q.なぜか羊がスポーンしません。

A.羊は芝生ブロックの上でしかスポーンしません。また芝生ブロックはある程度の明るい場所でしか伝播しません。
対処:松明・グロウストーン等の明かりを設置する。草ブロックを増やす。

**Comment by Orange2525** [comment]
Q.エンドポータルフレームが見つかりません。

A.エンドポータルフレームのない島もあります。設置されている場合、隠れていたとしても分かりやすい位置にあります。
